tests/smoke-pipenv.yaml

```diff
@@ -75,7 +75,7 @@ output:
             updated-dependency-files:
                 - content: |
                     [[source]]
-                    url = "https://pypi.python.org/simple"
+                    url = "https://pypi.python.org/simple"  # TODO: why isn't this auto-changing to pypi.org? Since it's a test output, not an input
```

Comment:
Something is weird here... I don't understand why these two values aren't changing? IIUC, these are test outputs, not inputs, and I've already changed manually the test inputs in `pipenv/Pipfile`, so that should result in this output file changing as well... 

Is there some cache that I'm unaware of?

Reproduction instructions are in the PR description...

Reply:
This was the solution: 
https://github.com/dependabot/smoke-tests/pull/114/files#diff-76182215302310a19532ad45c8f492ff4664a7086f28d11bc7c4bbe06b0e7b19R13

I didn't realize the inputs had a hardcoded reference to a particular commit that it was pulling the snapshot of the manifests from... once I updated that to point to a commit that included my updates to the manifests then it worked as expected.

🎩 💁‍♂️  @jakecoffman for pointing me toward this.